ROADMAP.md

```diff
@@ -2,23 +2,100 @@
 
 This document is meant to provide transparency in planning and implementation goals for the Koop project.
 
-## July 2015
-
-### Changes 
-* FeatureService optimizations 
-  * Make smarter requests to the DB based on queries 
-  * clean up the logic to be easier to grok
-* Refactor the lib/* files on a whole
-  - We want pull out as much non-koop specific code as possible into separate modules
-* Add support for the simple style spec - https://github.com/mapbox/simplestyle-spec
-* Address consistency issues in method calls to the Cache from providers 
-  - be clear and consitent about passing table names instead of "types" and "keys"
-
-## September 2015
-
-### Changes 
-* Explore removing ogr2ogr as a dep for exporting file formats
-### Additions 
-* Support for exporting FGDB
-* Support for OGC WMS and WFS 
-* A koop desktop app built with something like Electron
+## v3
+
+Major API changes:
+
+Provide better support for `plugins`. Right now the model for creating a plugin is pretty poor, as illustrated by https://github.com/koopjs/koop/issues/195.
+
+Move away from `provider` in favor of three new types of plugins: `source`, `export`, and `interface`.
```

Comment:
It is a bit far-reaching, but I think it's important to have a long-term plan for what an ideal Koop looks like rather than letting development be guided by feature creep.

With that said, I agree we should focus in the short-term on your above recommendations. Perhaps we can use that as a template for a v3 and call this v4 or vN.